### PR TITLE
OKTA-299645 Add .NET code samples to token validation guide

### PR DESCRIPTION
**Pull request by andriizhegurov-okta**
Initial commit

## Description:
- **What's changed?** 
Added instructions for .Net here `/docs/guides/validate-access-tokens/dotnet/overview/` -- taken from `/code/dotnet/jwt-validation/`
- **Is this PR related to a Monolith release?** 
No
### Resolves:

* [OKTA-299645 Add .NET code samples to token validation guide](https://oktainc.atlassian.net/browse/OKTA-299645)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/validate-access-tokens/overview/dotnet/accesstoken.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+if (!nonceMatches)
+{
+    throw new SecurityTokenValidationException("The nonce was invalid.");
+}
+```
+
+## Decode token claims
+
+The sample `ValidateToken` method above both validates a token and decodes its claims. You can use the returned `JwtSecurityToken` object to inspect the claims in the token.
```
Comment (bryanapellanes-okta) [inline packages/@okta/vuepress-site/docs/guides/validate-access-tokens/overview/dotnet/accesstoken.md:161]:
```suggestion
The sample `ValidateToken` method above both validates a token and decodes its claims. Use the returned `JwtSecurityToken` object to inspect the claims in the token.
```
Active voice.

File: packages/@okta/vuepress-site/docs/guides/validate-access-tokens/overview/dotnet/accesstoken.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+For example, you can get the `sub` (Subject) claim with the `Subject` property:
```
Comment (bryanapellanes-okta) [inline packages/@okta/vuepress-site/docs/guides/validate-access-tokens/overview/dotnet/accesstoken.md:163]:
```suggestion
For example, get the `sub` (Subject) claim with the `Subject` property:
```
Active voice.

File: packages/@okta/vuepress-site/docs/guides/validate-access-tokens/overview/dotnet/accesstoken.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+```csharp
+Console.WriteLine($"Token subject: {validatedToken.Subject}");
+```
+
+You can access more claims with the `Payload` property, or loop over the entire `Claims` collection:
```
Comment (bryanapellanes-okta) [inline packages/@okta/vuepress-site/docs/guides/validate-access-tokens/overview/dotnet/accesstoken.md:169]:
```suggestion
Access more claims with the `Payload` property, or loop over the entire `Claims` collection:
```
Active voice.

**Review by bryanapellanes-okta — COMMENTED** [review]
Some suggestions.

**Review by bryanapellanes-okta — APPROVED** [review]
Approving, considering this is in line with existing docs.

**Comment by andriizhegurov-okta** [comment]
@jakubvul-okta
cc @laura-rodriguez 
In this PR I remove unreachable article from there `/code/dotnet/jwt-validation/` and put it here `/docs/guides/validate-access-tokens/dotnet/overview/` where all the others JWT verifiers are.
Will you approve the deletion of this article in the original place?

**Comment by andriizhegurov-okta** [comment]
@jakubvul-okta 
cc @laura-rodriguez
I restored the deleted article because it is referenced from three other pages.

**Pull request merged**